packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/TypeProvider.cs

```diff
@@ -423,13 +448,39 @@ private bool ShouldGenerate(PropertyProvider property, HashSet<string> customPro
             return !customProperties.Contains(property.Name);
         }
 
+        private bool ShouldGenerate(FieldProvider field, HashSet<string> customFields)
+        {
+            // partial fixed enums are not supported so we don't need to check for custom fields in enums
+            if (field.EnclosingType?.IsEnum == true)
```

Comment:
Couldn't this be an extensible enum?

Reply:
Good point. I explicitly restricted this to fixed enums now.